incubator/hnc/internal/forest/namespace.go

```diff
@@ -27,6 +27,10 @@ type Namespace struct {
 	exists               bool
 	allowCascadingDelete bool
 
+	// labels store the original namespaces' labels, and we add the propogate label to identify itself
```

Comment:
Update the comment since we don't have propagate labels now after the first push.

Reply:
Done